sp800-63b/sec5_authenticators.md

```diff
@@ -314,21 +316,19 @@ CSPs SHALL provide subscriber instructions on how to appropriately  protect the
 
 #### <a name="throttle"></a>5.2.2. Rate Limiting (Throttling)
 
-*cf. 800-63-2 sec 8.2.3, p.75*
-
-When the authenticator output or activation secret does not have sufficient entropy, the verifier SHALL implement controls to protect against online guessing attacks. Unless otherwise specified in the description of a given authenticator, the verifier SHALL effectively limit online attackers to 100 consecutive failed attempts on a single account in any 30 day period.
+When the authenticator output or activation secret does not have sufficient entropy, the verifier SHALL implement controls to protect against online guessing attacks. Unless otherwise specified in the description of a given authenticator, the verifier SHALL effectively limit online attackers to 100 consecutive failed attempts on a single account.
```

Comment:
Well, we lost the 30 days too for some reason, so I don't know what the heck is going on with this.  I thought we were all good with 30 days and 'at most...'.  I would like that back in.  I think that means rejecting Mike's edit.